src/DetectChanges/BCBreak/EnumBased/EnumBased.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Roave\BackwardCompatibility\DetectChanges\BCBreak\EnumBased;
+
+use Roave\BackwardCompatibility\Changes;
+use Roave\BetterReflection\Reflection\ReflectionClass;
+
+interface EnumBased
```

Comment:
While this interface makes sense, I would make it extend from `ClassBased`, since ENUMs in PHP are just `final` classes

Reply:
If I remove the change to CompareClasses as you suggested and I'm planning to do then I'd just delete this interface and use ClassBased. It doesn't have any significant distinction.

The only difference between this and ClassBased is the different param names, which I think would mean SA tools will rightly say it's not allowed to extend ClassBased.

Reply:
Interface removed.